### PR TITLE
Enabling Currents.dev on CircleCI

### PR DESCRIPTION
**Pull request by diogormendes**
This PR enables the Cypress tests to send their results to  [Currents.dev](http://currents.dev/)

Screen Shot 2022-03-17 at 17 24 30

Link to run: https://app.currents.dev/run/b2a895aadb5bf8597ca63c65bd8817a0

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -943,9 +946,12 @@ jobs:
                       name: Restore cached uberjar built in previous step
                       <<: *CacheKeyUberjar
                   - steps: << parameters.before-steps >>
-                # Make both `test-files` and `source-folder` parameters optional. Translates to: if `parameter` => run associated flag (`--spec` and `--folder`, respectively)
+                # Make both `test-files`,  `source-folder` and `currents-record` parameters optional. Translates to: if `parameter` => run associated flag (`--spec`, `--folder` and `--key $CURRENTS_KEY --record` respectively)
                 command: |
-                  run test-cypress-no-build <<# parameters.test-files >> --spec << parameters.test-files >> <</ parameters.test-files >> <<# parameters.source-folder >> --folder << parameters.source-folder >> <</ parameters.source-folder >>
+                  run test-cypress-no-build \
+                  <<# parameters.test-files >> --spec << parameters.test-files >> <</ parameters.test-files >> \
+                  <<# parameters.currents-record >> --key $CURRENTS_KEY --record << parameters.currents-record >> <</ parameters.currents-record >> \
```
Comment (nemanjaglumac) [inline .circleci/config.yml:953]: `--record` flag is enough. It doesn't need additional parameters.

```suggestion
                  <<# parameters.currents-record >> --key $CURRENTS_KEY --record <</ parameters.currents-record >> \
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -943,9 +946,12 @@ jobs:
                       name: Restore cached uberjar built in previous step
                       <<: *CacheKeyUberjar
                   - steps: << parameters.before-steps >>
-                # Make both `test-files` and `source-folder` parameters optional. Translates to: if `parameter` => run associated flag (`--spec` and `--folder`, respectively)
+                # Make both `test-files`,  `source-folder` and `currents-record` parameters optional. Translates to: if `parameter` => run associated flag (`--spec`, `--folder` and `--key $CURRENTS_KEY --record` respectively)
                 command: |
-                  run test-cypress-no-build <<# parameters.test-files >> --spec << parameters.test-files >> <</ parameters.test-files >> <<# parameters.source-folder >> --folder << parameters.source-folder >> <</ parameters.source-folder >>
+                  run test-cypress-no-build \
```
Comment (nemanjaglumac) [inline .circleci/config.yml:951]: Are you sure we need backslashes at the end of each row?
```suggestion
                  run test-cypress-no-build
```

**Review by nemanjaglumac — APPROVED** [review]
Looks much better now 💪 

Left a few comments and a nit regarding the use of backslash. IIRC, the multi-line command works fine with out them. But if it works like this, it doesn't really matter.

File: .circleci/config.yml (hunk 2 above)
Comment (nemanjaglumac) [reply .circleci/config.yml:951]: But isn't the pipe next to command exactly the pointer that says: "Hey, interpret next lines as multi-line command"?
`command: |`

Comment (diogormendes) [reply .circleci/config.yml:951]: As I understand, every time we run `command` CCI runs on a shell. 

In a shell script, if we are doing multiple lines in one command we need to properly add the `\` to it to understand that the command continues. If not, it understands as one command per line.

**Pull request merged**